HousingSearchApi/V1/Infrastructure/ElasticElasticSearchWrapper.cs

```diff
@@ -121,10 +122,60 @@ public class ElasticSearchWrapper : IElasticSearchWrapper
             }
 
         }
+        public async Task<ISearchResponse<QueryableTenure>> SearchTenuresSets(GetAllTenureListRequest request)
+        {
+            var esNodes = string.Join(';', _esClient.ConnectionSettings.ConnectionPool.Nodes.Select(x => x.Uri));
+
+            _logger.LogDebug($"ElasticSearch Search Tenures Sets begins {esNodes}");
+
+            if (request == null)
+            {
+                return new SearchResponse<QueryableTenure>();
+            }
+
+            //last hit paging is only supported when sorting by tenure start date
+            //any other type of paging is not supported. Page size can be set however when using start date sorting with last hit paging
```

Comment:
That's a good point about the note. I've updated the end point documentation in [#1618d3b](https://github.com/LBHackney-IT/housing-search-api/pull/226/commits/1618d3b60e5fb9d4d82b2ccbd9b9d57200792f25).

As for the functionality I was thinking about the same thing. However there's nothing TA specific here really since only enhancement is just the ability to be able to sort by tenureStartDate while using last hit id style paging. This could apply to other than TA use cases. For that reason I left it open for now. The way ES handles paging and sorting in that scenario means that it takes a bit of work to cover different use cases, say sorting by different or multiple properties, so this uptate only covers the bare minimum to achieve the functionality we need for the TA worktray. I did make the decision to not make  the gateway method generic for example, so it's tightly coupled to tenures, so there is some locking down happening already. That's not to say that it couldn't be written using generics, but it would required considerable amount of work and since we don't have any other use cases in mind right now I didn't think it was wort it.